src/app-layout/__integ__/app-layout-refresh-notifications.test.ts

```diff
@@ -30,11 +30,11 @@ class AppLayoutRefreshNotoficationsPage extends BasePageObject {
   async assertNotificationsWidth(comparison = 0) {
     const contentWidth = await this.getContentWidth();
     if (comparison < 0) {
-      expect(this.getNotificationsWidth()).resolves.toBeLessThan(contentWidth);
+      await expect(this.getNotificationsWidth()).resolves.toBeLessThan(contentWidth);
```

Comment:
Well, that's unfortunate. Hard to believe that many tests were flaky and we did not notice it.

Thanks for the PR, very useful!

Reply:
Yes 😱 Glad I added it now!